src/presolve.jl

```diff
@@ -37,7 +35,7 @@ end
 
 This function implements the bound-tightening algorithm to tighten the variable bounds.
 It utilizes either the basic McCormick relaxation or the Tightened McCormick relaxation (TMC)
-to tighten the bounds. The TMC has additional binary variables for partitioning.
+to tighten the bounds. The TMC has additional binary variablers for partitioning.
```

Comment:
you corrected a correct spelling and made it incorrect - variables

Reply:
Sorry about that.